pkg/restapi/issuer/operation/operations.go

```diff
@@ -118,6 +120,7 @@ type connections interface {
 // PublicDIDCreator creates public DIDs.
 type PublicDIDCreator interface {
 	Create() (*did.Doc, error)
+	CreateV2() (*did.Doc, error)
```

Comment:
What are the DID document properties differ between v1 and v2 ? If its few attributes differ, then can't we pass those as function args ?

Reply:
@rolsonquadras I tried this, and it didn't make the code any simpler, so I'll just leave it like this.